unused/Battle_of_Jingxing.md

```diff
@@ -0,0 +1,8 @@
+The Battle of Jingxing was fought in 205 BC between forces of the Han kingdom and those of the Zhao. The Han numbered 30,000 and were commanded by Han Xin. The Zhao led by commander Chen Xu were, with their 200,000 men, not only numerically superior but also, since the Han army consisted entirely of peasant militia, more experienced. This was because Han Xin, the Han commander, had been obligated to send his best man to his warlord.
+The two armies faced each other in the Jingxing Pass, leading across the Taihang Mountains and into the Zhao kingdom. Because this narrow pass was perfectly suited for an ambush, Han Xin was very cautious about entering it and did so only after careful scouting. And the Zhao were indeed not planning an ambush; they held their higher manpower alone for a big enough advantage. Instead, they camped outside the pass, waiting for the Han.
```

Comment:
```suggestion
The two armies faced each other in the Jingxing Pass, which lead across the Taihang Mountains and into the Zhao kingdom. Because this narrow pass was perfectly suited for an ambush, Han Xin was very cautious about entering it and did so only after careful scouting. And the Zhao were indeed not planning an ambush; they held their higher manpower alone for a big enough advantage. Instead, they camped outside the pass, waiting for the Han.
```